### PR TITLE
tc for Firehose and Redshift data validation were added

### PR DESCRIPTION
**Pull request by rohitjain-bidgely**
Execution_Run
Attached the Execution run as well.

1)Ingestion class should not have both multimeter and single meter test cases otherwise not able to get the unique uuid and firehose and redshift validation can not able to run.

### CODE REVIEW

File: src/main/java/constants/UserType.java
Comment (akhil-bidgely) [inline src/main/java/constants/UserType.java:1]: Insert these constants in existing constatns files, no need to create a seperate file.

File: src/main/java/constants/S3Constants.java
Comment (akhil-bidgely) [inline src/main/java/constants/S3Constants.java:1]: Insert these constants in existing files, no need to create a seperate file.

File: src/main/java/serviceHelper/RestUtils.java
Comment (akhil-bidgely) [inline src/main/java/serviceHelper/RestUtils.java:1]: Change the convention to Rest specs as used in other methods

File: src/test/java/responseValidation/IngestionValidations.java
Comment (akhil-bidgely) [inline src/test/java/responseValidation/IngestionValidations.java:1]: Add reporting for all the validations.

File: src/test/java/tests/Ingestion.java
Comment (akhil-bidgely) [inline src/test/java/tests/Ingestion.java:1]: Remove static file from filePath

File: src/main/java/constants/UserType.java
Comment (rohitjain-bidgely) [reply src/main/java/constants/UserType.java:1]: done

File: src/main/java/constants/S3Constants.java
Comment (rohitjain-bidgely) [reply src/main/java/constants/S3Constants.java:1]: done

File: src/main/java/serviceHelper/RestUtils.java
Comment (rohitjain-bidgely) [reply src/main/java/serviceHelper/RestUtils.java:1]: done

File: src/test/java/responseValidation/IngestionValidations.java
Comment (rohitjain-bidgely) [reply src/test/java/responseValidation/IngestionValidations.java:1]: have to work on it separately.

File: src/test/java/tests/Ingestion.java
Comment (rohitjain-bidgely) [reply src/test/java/tests/Ingestion.java:1]: done

**Comment by rohitjain-bidgely** [comment]
following is the screen shot of tc's execution
Screenshot 2023-05-16 at 11 41 16 AM

**Review by akhil-bidgely — APPROVED** [review]

**Pull request merged**